developer-hub/src/polkadot_sdk/polkadot.rs

```diff
@@ -0,0 +1,82 @@
+//! # Polkadot
+//!
+//! Implementation of the Polkadot node/host in Rust.
+//!
+//! ## Learn More and Get Involved
+//!
+//! - [Polkadot Forum](https://forum.polkadot.network/)
+//! - [Polkadot Parachains](https://parachains.info/)
+//! - [Polkadot (multi-chain) Explorer](https://subscan.io/)
+//! - Polkadot Fellowship
+//!     - [Manifesto](https://github.com/polkadot-fellows/manifesto)
+//!     - [Runtimes](https://github.com/polkadot-fellows/runtimes)
+//!     - [RFCs](https://github.com/polkadot-fellows/rfcs)
+//! - [Polkadot Specs](spec.polkadot.network)
+//! - [The Polkadot Parachain Host Implementers' Guide](https://paritytech.github.io/polkadot-sdk/book/)
+//! - [Whitepaper](https://www.polkadot.network/whitepaper/)
+//!
+//! ## Alternative Node Implementations 🌈
+//!
+//! - [Smoldot](https://crates.io/crates/smoldot-light). Polkadot light node/client.
+//! - https://github.com/qdrvm/kagome
```

Comment:
Should probably follow the same convention as the other links? e.g.

```suggestion
//! - [KAGOME is a C++ implementation of the Polkadot host](https://github.com/qdrvm/kagome)
```